app.rb

```diff
@@ -0,0 +1,29 @@
+require 'sinatra/base'
+require_relative 'lib/database_connection_setup'
+require_relative 'lib/peep'
+
+class Chitter < Sinatra::Base
+
+  get '/' do
+    @peeps = Peep.all
+
+    erb :index
+  end
+
+  post '/' do
+    @peeps = Peep.all
+    message = params['message']
+
+    @zero_length_msg = message.length.zero?
```

Comment:
this maybe could have been in a ./lib/?.rb file as a separate method to keep the app.rb clean from logic. It's ok with shorter codes but it would be easier to understand longer codes in the future.

Reply:
Agreed! I really should make time for refactoring in the weekend challenges...